src/components/shareButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+import { SizeType } from 'antd/es/config-provider/SizeContext';
+import ShareMenu from '../shareMenu';
+import ShareWhite from '../../assets/images/sharewhite-small.png';
```

Comment:
nit: I personally find the other share icons more intuitive (arrow to top right from window, 3 node tree) and didn't know what this button meant at first. It might just be me though.

Reply:
This one is on the more abstract side, but it's also not uncommon - I believe Instagram and Twitter use this icon, just to name a few, so there's a good chance people have seen it before. To me, the arrow implies a reply, although that doesn't make sense in this context, and I haven't seen the 3-node tree around that much lately - I think really any of these could work, though

Reply:
Yup, that works then.